src/common/harvest/harvest.js

```diff
@@ -180,7 +180,8 @@ export class Harvest extends SharedContext {
       '&ck=0', // ck param DEPRECATED - still expected by backend
       '&s=' + (runtime.session?.state.value || '0'), // the 0 id encaps all untrackable and default traffic
       encodeParam('ref', ref),
-      encodeParam('ptid', (runtime.ptid ? '' + runtime.ptid : ''))
+      encodeParam('ptid', (runtime.ptid ? '' + runtime.ptid : '')),
+      encodeParam('hr', (runtime?.session?.state.sessionReplayMode === 1 ? '1' : '0'), qs) // hasReplay
```

Comment:
So only if we are in full session replay?

Reply:
yep